### PR TITLE
Readme: Add docu about status HTTP endpoint

### PR DESCRIPTION
**Pull request by vogdb**
@tomaszdurka @vogdb add some comments / maybe curl example?

### CODE REVIEW

**Comment by vogdb** [comment]
@tomaszdurka please review.

File: README.md [hunk 1]
```diff
@@ -64,6 +64,32 @@ jobManager:
...
+    - streamId {String}. required. Id of stream to stop.
+ - method: POST.
+ - Response: `{success: 'Stream stopped'}` or `{error: '<reason>'}`.
+ - Example curl request:
+
+    ```
+    curl -H "Server-Key:yourSecretKey" -X POST -d "streamId=12312" http://localhost:8888/stopStream
+    ```
+
+### /status.
```
Comment (tomaszdurka) [inline README.md]: Remove the `dot`!

**Comment by vogdb** [comment]
@tomaszdurka please review

**Comment by tomaszdurka** [comment]
lgtm

**Pull request merged**